### ISSUE TITLE
Neovim transition on Win10, Vundle

### ISSUE THREAD

**Issue by alekulyn** [neovim/neovim#3747]
I transformed to Neovim a few days ago on Win10 64bit, and I copied over my vimrc and vimfiles/* to %LOCALAPPDATA%/nvim.  I noticed that, although all my plugins were working in vim, all the plugin commands I ran (PluginInstall, PluginClean, PluginList) were triggering this error.
![Nvim_Error](https://cloud.githubusercontent.com/assets/3824241/11213739/541a6b3e-8d46-11e5-9801-6e3c08bb5a00.png)

When I open the logs, this is what pasted:
![Error_log](https://cloud.githubusercontent.com/assets/3824241/11435054/9d1a903a-94de-11e5-97e8-52558630856a.png)

This is a copy of the relevant parts of my init.vim (Neovim's vimrc):
```vim
let vundle_path=$LOCALAPPDATA.'/nvim/bundle'
set rtp+=$LOCALAPPDATA/nvim/bundle/Vundle.vim/
call vundle#begin(vundle_path)

" let Vundle manage Vundle, required
Plugin 'VundleVim/Vundle.vim'

"Plugin 'scrooloose/nerdtree'
"Plugin 'kien/ctrlp.vim'
"Plugin 'scrooloose/syntastic'
"Plugin 'bling/vim-airline'
"Plugin 'tpope/vim-fugitive'
"Plugin 'nathanaelkane/vim-indent-guides'
"Plugin 'xolox/vim-session'
"Plugin 'Valloric/YouCompleteMe'
```

The other plugins are commented out because I'd like to get Vundle working first.  I already posted this issue in the Vundle issue tracker (https://github.com/VundleVim/Vundle.vim/issues/670), but they told me it seemed to be a neovim problem.

**Comment by justinmk** [comment]
Windows support is experimental, currently. There are probably many bugs.

**Comment by equalsraf** [comment]
The read only errors may be related to #3526, now that #3531 was merged, maybe the issue is fixed (I did not get the chance to test yet, but 810 has been rebased)

**Comment by justinmk** [comment]
Thanks @equalsraf

**Comment by alekulyn** [comment]
Thankfully, the read-only errors are gone, but the second error log is still there.  Am I correct in guessing that this is not a Neovim problem anymore?

**Comment by justinmk** [comment]
Don't know. Not enough information.

**Comment by equalsraf** [comment]
@alekulyn I have not checked vundle internals, wild guess would be some problem with path escaping, there is a known issue in system() escaping, so maybe that is the problem

**Comment by justinmk** [comment]
Similar issue with vim-plug, so it's a Neovim bug.

**Comment by equalsraf** [comment]
This should be fixed in #810 now.
